frontend/components/WaitingPages/HostWaiting.tsx

```diff
@@ -56,115 +88,138 @@ export default function HostWaiting({ hostId }: hostProps) {
+											+
+										</button>
+									</div>
+									<div className="flex flex-row bg-gray-100 justify-between items-center rounded-xl border-2 border-gray-200 mb-2 mx-2 w-28 h-11">
+										<button
+											className="bg-white border-1 border-gray-200 rounded text-red-400 text-center w-7 h-8 mx-1"
+											onClick={() => {
+												decrementPlayers();
+											}}
+										>
+											-
+										</button>
+										<div className="text-lg">{maxPlayers}</div>
+										<button
+											className="bg-white border-1 border-gray-200 rounded text-green-400 text-center w-7 h-8 mx-1"
+											onClick={() => {
+												incrementPlayers();
+											}}
+										>
+											+
+										</button>
+									</div>
+								</div>
+							</div>
 						</div>
-						<input
-							className="w-32 mb-2"
-							id="maxPlayers"
-							type="text"
-							maxLength={4}
-							onChange={handlePlayerChange}
-						/>
 					</div>
 				</div>
 			</div>
-		</div>
-	);
-
-	const desktopContent = (
-		<div className="flex flex-row items-center justify-start">
-			<div className="bg-gray-100 flex flex-row items-center justify-between font-extrabold shadow-heavy rounded-xl p-6 w-min">
-				<div>
-					<div className="text-4xl xl:text-5xl whitespace-nowrap">
-						Join with the code:
+			<div className="hidden sm:block w-max lg:w-full h-full py-4">
+				<div className="absolute rounded-xl shadow-heavy bg-purple-50 text-4xl font-extrabold py-4 px-6 w-fit up-2">
+					Participants
+				</div>
+				<div className="flex flex-col w-full h-full shadow-heavy rounded-xl bg-gray-100 bg-opacity-50 my-4 z-0">
+					<div className="w-full h-full p-4">
+						<table className="w-full border-separate border-spacing-y-3 text-2xl font-extrabold mt-14">
+							{displayPlayers.map((subArray, index) => (
+								<tr className="border-separate border-spacing-y-2" key={index}>
+									{subArray.map((player, i) => (
+										<td key={index + i}>
+											<div
+												className={`text-white text-center w-52 rounded-xl py-1 ${
+													colors[(i + index) % 5]
+												}`}
+											>
+												{player}
+											</div>
+										</td>
+									))}
+								</tr>
+							))}
+						</table>
+					</div>
+					<div className="flex flex-row justify-end m-2">
+						<div className="rounded-xl bg-purple-100 text-2xl w-52 text-white text-center m-2 px-8 py-2">
+							{playerList.length}/{maxPlayers}
+						</div>
 					</div>
-					<div className="text-8xl xl:text-9xl">{hostId}</div>
 				</div>
-				{headerContent}
+			</div>
+			<div className="flex flex-row justify-center items-center mt-4">
 				<button
-					className="bg-orange-200 hover:bg-orange-100 border-1 border-gray-200 rounded-xl mx-2 text-white text-center text-4xl xl:text-5xl shadow-heavy w-48 h-20"
+					className="bg-purple-50 hover:bg-gray-100 border-1 border-gray-200 rounded-xl mx-2 text-white text-center text-4xl shadow-heavy w-1/4 h-24"
```

Comment:
The text here is a bit large on small screens, so you could adjust this and the start button text to be text-2xl or something similar until the screen is larger. I also suspect that the fixed height here is also contributing to the buttons overflowing vertically on smaller screens; it might be better to use proportions if possible, like with the width.